builder/bin/image_gen.rs

```diff
@@ -53,7 +62,27 @@ fn main() {
             },
         )
         .unwrap();
-        std::fs::write(path, image.to_bytes().unwrap()).unwrap();
+
+        let contents = image.to_bytes().unwrap();
+        std::fs::write(path, contents.clone()).unwrap();
+
+        if let Some(path) = args.get_one::<PathBuf>("hashes") {
+            let header_range = span_of!(ImageManifest, header);
+
+            // Get the vendor digest which is taken from a subset of the header
+            let vendor_header_len = offset_of!(ImageHeader, owner_data);
+            let vendor_range = header_range.start..header_range.start + vendor_header_len;
+            let vendor_digest = Sha384::digest(&contents[vendor_range]);
+
+            // Get the owner digest which is the full header
+            let owner_digest = Sha384::digest(&contents[header_range]);
+
+            let json = json!({
+                "vendor": vendor_digest.to_vec(),
+                "owner": owner_digest.to_vec(),
```

Comment:
Should these vec's be encoded into a string representation, like base64?

Reply:
I don't have a really strong opinion. I could see a hexstring being an easily readable and consumable type as well. @jhand2 what do you think?

Reply:
I think my preference would be hex string

Reply:
The output file now looks like this:

```json
{
  "owner": "a506d220c5f9023a3fdeed8ca89d2da35219e20cef6a1d5c0143bff6e763059b62a4a43582dfb6903d50a98d0c99660e",
  "vendor": "0a4a24ab60381a5052daffdfab3fceaa47f793ccab62fe30841e842121b5da917e95b8ae5c08924498308c75f87882bb"
}
```